packages/postgres-database/src/simcore_postgres_database/models/payments_transactions.py

```diff
@@ -0,0 +1,110 @@
+import sqlalchemy as sa
+
+from ._common import (
+    column_created_datetime,
+    column_modified_datetime,
+    register_modified_datetime_auto_update_trigger,
+)
+from .base import metadata
+
+payments_transactions = sa.Table(
+    "payments_transactions",
+    metadata,
+    sa.Column(
+        "payment_id",
+        sa.String,
+        nullable=False,
+        primary_key=True,
+        doc="Identifer of the payment provided by payment gateway",
+    ),
+    sa.Column(
+        "price_dollars",
+        sa.Numeric(scale=2),
+        nullable=False,
+        doc="Total amount of the transaction (in dollars). E.g. 1234.12 $",
```

Comment:
same question here. we are here using dollars. this will remain so? no euro, or other currencies foreseen?

Reply:
Same answer above